index.html

```diff
@@ -0,0 +1,142 @@
+      <a href="about.html" class="toggle1"><h5 class="red-color">About</h5></a>
+      <a href="#program" class="toggle1"><h5>Program</h5></a>
+      <a href="#join" class="toggle1"><h5>Join</h5></a>
+      <a href="#sponsor" class="toggle1"><h5>Sponsor</h5></a>
+      <a href="" class="toggle1"><h5>News</h5></a>
+      <a href="" class="nav-bar-h56 toggle1"><h5 class="red-color">SC Campaign</h5></a>
+    </nav>
+    
+  </header>
+
+  <section class="intro-text">
+    <h3>"Hello! Sharing world"</h3>
+    <h1>SAVE CAUVERY 2022</h1>
+    <p id="para-border">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Esse voluptas quas magnam reprehenderit incidunt! Similique optio in aliquid autem aut voluptate velit nihil iure molestias!</p>
+    <p id="para-time">2022.11.21(THU) ~ 22(FRI)</p>
+    <p id="para-place">@ Cauvery River Karnataka, Tamilnadu</p>
+  </section>
+
+  <section class="main-program" id="program">
+    <h2 id="main-program-header">Main Program</h2>
+    <hr id="main-program-hr"/>
+    <div class="main-program-container">
+      <div class="main-program-item">
+        <img src="Images/mpi-icon1.png">
+        <h3>Lecture</h3>
+        <p class="mpi-p">Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempore fuga odio blanditiis?</p>
+      </div>
+      <div class="main-program-item">
+        <img src="Images/mpi-icon2.png">
+        <h3>Lecture</h3>
+        <p class="mpi-p">Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempore fuga odio blanditiis?</p>
+      </div>
+      <div class="main-program-item">
+        <img src="Images/mpi-icon3.png">
+        <h3>Lecture</h3>
+        <p class="mpi-p">Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempore fuga odio blanditiis?</p>
+      </div>
+      <div class="main-program-item">
+        <img src="Images/mpi-icon4.png">
+        <h3>Lecture</h3>
+        <p class="mpi-p">Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempore fuga odio blanditiis?</p>
+      </div>
+      <div class="main-program-item">
+        <img src="Images/mpi-icon5.png">
+        <h3>Lecture</h3>
+        <p class="mpi-p">Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempore fuga odio blanditiis?</p>
+      </div>
+    </div>
+    <button class="main-program-button">Join Save Cauvery 2022</button>
+    <h3 class="mi-hidden">SEE THE WHOLE PROGRAM</h3>
+  </section>
+
+  <section class="speakers" id="join">
+    <h3 class="speaker-h31">Featured Speakers</h3>
+    <hr class="speaker-hr">
+    <div class="speaker-container">
+    </div>
+    <button class="speaker-button">MORE <img class="button-image" src="Images/button-img.png"></button>
+</section>
+
+  <section class="partners" id="sponsor">
+    <h3>Partner</h3>
+    <hr id="body-hr3">
+    <div class="container text-center partners-mobile">
+      <div class="row" id="row1">
+        <div class="col">
```

Comment:
- [ ] In the mobile design of the home page there isn't a partner section. Kindly fix this 🙏